### ISSUE TITLE
Entity rotation setter is not working as intended

### ISSUE THREAD

**Issue by Lekenthun** [altmp/altv-issues#171]
It would be tremendously helpful if vehicle.rot.property worked as setter as well. It is usually needed, f.e.: for unflip command.

**Comment by Lekenthun** [comment]
The vehicle.rot setter actually works, but it only rotates the vehicle to the north for the users who have streamed the car in. if we leave the streaming distance and go back, then the car will be in the right direction

**Comment by c0nnex** [comment]
Since i run into this problem as well maybe I can shed some light on the issue.
I made some quick tests, and the rotation is being wrong on the server even if no client is connected. 
I found no way to have the server have the right rotation other than checking the syncer and enforcing the syncer to set the rotation on client, to update it. 
For I endend up using my own synced rotation, which is really lame imho.

// Even with no client online the rotation will be wrong
import * as alt from 'alt-server';

let veh = new alt.Vehicle(2364918497, -894.8011, -2620.8477, 16.191927, 23,255, 59);
let rot = new alt.Vector3(0,0,0);
alt.everyTick( () => {
  if (veh != null)
  {
    let r = veh.rot;
	if (r.z != rot.z)
	{
	   alt.log(`rot changed old ${rot} new ${r} pos ${veh.pos}`);
	   rot = r;
    }
  }
});
